mail_tracking/models/mail_tracking_email.py

```diff
@@ -77,13 +84,81 @@ class MailTrackingEmail(models.Model):
         string="Tracking events", comodel_name='mail.tracking.event',
         inverse_name='tracking_email_id', readonly=True)
 
+    @api.model
+    def email_score(self, trackings):
+        """Default email score algorimth"""
```

Comment:
This is for reusing this in other models (like crm.lead) and inherit only in one place